org.gazebosim.Gazebo.yaml

```diff
@@ -0,0 +1,675 @@
+  - shared-modules/glu/glu-9.json
+  - shared-modules/libusb/libusb.json
+  
+  - name: libjxr
+    buildsystem: cmake-ninja
+    build-options:
+      cflags: -fPIC
+      cxxflags: -fPIC
+    sources:
+      - type: archive
+        url: https://github.com/4creators/jxrlib/archive/refs/tags/v2019.10.9.tar.gz
+        sha256: 555c006e27c5cb66f99c05dcbb2feb197199ca9018dbd06d3467d37cd29a79cd
+      - type: file
+        path: JXR-CMakeLists.txt
+        dest-filename: CMakeLists.txt
+      
+  - name: LibRaw
+    buildsystem: simple
+    build-commands:
+      - autoreconf -vfi
+      - ./configure --disable-example --enable-lcms --prefix=/app
+      - make
+      - make install
+    sources:
+      - type: archive
+        url: https://www.libraw.org/data/LibRaw-0.20.2.tar.gz
+        sha256: dc1b486c2003435733043e4e05273477326e51c3ea554c6864a4eafaff1004a6
+    cleanup: 
+      - /bin
+      - /share/doc
+        
+  - name: Imath
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DBUILD_TESTING:BOOL=OFF
+    sources:
+      - type: git
+        url: https://github.com/AcademySoftwareFoundation/Imath.git
+        tag: v3.0.1
+  
+  - name: openEXR
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DBUILD_TESTING:BOOL=OFF
+    sources:
+      - type: git
+        url: https://github.com/AcademySoftwareFoundation/openexr.git
+        tag: v3.0.1
+        
+  #- name: libwebp
+    #buildsystem: cmake-ninja
+    #build-options:
+      #cflags: -fPIC
+      #cxxflags: -fPIC
+    #sources:
+      #- type: git
+        #url: https://chromium.googlesource.com/webm/libwebp/
+        #tag: v1.2.0
+        
+  - name: FreeImage
+    build-options:
+      cflags: -I/app/include/jxrlib -I/app/include -I/usr/include -fpermissive -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -fPIC -Wunused-variable
+      cxxflags: -I/app/include/jxrlib -I/app/include -I/usr/include -fpermissive -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -fPIC -Wunused-variable
+    buildsystem: simple
+    build-commands:
+      - rm -rf Source/LibPNG/ Source/LibRawLite/ Source/OpenEXR/ Source/ZLib/ Source/LibOpenJPEG/ Source/LibJPEG/ 
+      - truncate -s 0 Source/FreeImageToolkit/JPEGTransform.cpp 
+      - truncate -s 0 Source/FreeImage/PluginG3.cpp
+      - for file in `find . -type f -name '*.c' -or -name '*.cpp' -or -name '*.h' -or -name '*.txt' -or -name Makefile`; do iconv --from=ISO-8859-15 --to=UTF-8 $file > $file.new && sed -i 's|\r||g' $file.new && touch -r $file $file.new && mv $file.new $file;
+        done
+      - sh ./gensrclist.sh
+      - sh ./genfipsrclist.sh
+      - make -f Makefile.gnu
+      - make -f Makefile.fip
+      - make install
+      - make -f Makefile.fip DESTDIR=/app INSTALLDIR=/app/lib install
```

Comment:
I found why it can't find the icon:

Here you need to add a cleanup:
````yaml
    cleanup:
      - /usr
````

appstream-compose fail because it goes to look up for icons in `/app/usr/share`... because there is `/app/usr/include` thanks to FreeImage.

There is an issue about that: 
https://github.com/hughsie/appstream-glib/issues/271

Reply:
worse. I had it mentionned to me a couple of days ago:

https://github.com/flathub/flathub/pull/2342#discussion_r634417363

But I didn't look at the presence of `/app/usr`

Reply:
Also this issue https://github.com/flatpak/flatpak-builder/issues/228

Reply:
Another alternative, UNTESTED, is to set `INCDIR=/app/include` and no set `DESTDIR` with the make install phase.

However I have only tested the suggestion above with `cleanup`

Reply:
Thank so much for the help. Definitely will keep this in mind.

Reply:
Thanks for your time.